frontend/javascripts/oxalis/constants.js

```diff
@@ -97,17 +97,46 @@ export const OrthoViewCrosshairColors: OrthoViewMap<[number, number]> = {
 };
 
 export const BorderTabs = {
-  DatasetInfoTabView: { id: "DatasetInfoTabView", name: "Info" },
-  CommentTabView: { id: "CommentTabView", name: "Comments" },
-  MeshesView: { id: "MeshesView", name: "Meshes" },
-  SkeletonTabView: { id: "SkeletonTabView", name: "Skeleton" },
-  AbstractTreeTab: { id: "AbstractTreeTab", name: "AbsTree" },
+  DatasetInfoTabView: {
+    id: "DatasetInfoTabView",
+    name: "Info",
+    description: "Information about the dataset",
+  },
+  CommentTabView: {
+    id: "CommentTabView",
+    name: "Comments",
+    description: "Add comments to skeleton nodes",
+  },
+  MeshesView: {
+    id: "MeshesView",
+    name: "Meshes",
+    description: "Load, compute and organize Meshes of segments",
+  },
+  SkeletonTabView: {
+    id: "SkeletonTabView",
+    name: "Skeleton",
+    description: "Create and organize skeletons",
+  },
+  AbstractTreeTab: {
+    id: "AbstractTreeTab",
+    name: "AbsTree",
+    description: "Views an abstract version of the active skeleton",
```

Comment:
```suggestion
    description: "Shows an abstract version of the active skeleton",
```

I think, "to view" would be the "act of looking" (which is done by the user and not be the tab).